Dockerfile

```diff
@@ -10,8 +10,17 @@ RUN { \
 WORKDIR /usr/src/app
 COPY . .
 
+RUN rm -r Gemfile.lock .ruby-lsp .git .github .DS_Store
+# If we're removing the Gemfile.lock we may as well remove rubocop, it's
+# a dev dep and won't affect tests or production
+RUN sed -i '/spec\.add_development_dependency "rubocop"/d' oas_agent.gemspec
```

Comment:
We could move rubocop into a group on it's own in the Gemfile and then run `bundle install --without lint` to ignore it here.

Reply:
We can remove this line now 😀